br: update some faq from oncall (#5411)
pingcap/docs: br/backup-and-restore-use-cases.md

### Preparation for backup

> The BR tool already supports self-adaptive GC, and automatically registers `backupTS` (the latest PD timestamp by default) to PD's `safePoint`. It ensures that TiDB's GC Safe Point does not move forward during the backup, thus avoiding the manual setting of GC.

Reviewer: ```suggestion
The BR tool already supports self-adapting to GC. It automatically registers `backupTS` (the latest PD timestamp by default) to PD's `safePoint` to ensure that TiDB's GC Safe Point does not move forward during the backup, thus avoiding manually setting GC configurations.
```
